### ISSUE TITLE
validation top1 error

### ISSUE THREAD

**Issue by Itsanewday** [liuzhuang13/DenseNet#43]
Hi
I trained a densenet according the paper on cifar10.The top1 error of test set is odd.  The result is saved in this file.
[net-train.pdf](https://github.com/liuzhuang13/DenseNet/files/1618831/net-train.pdf)
How can i make the top1 error curve smooth ? 
Thanks

**Comment by Sirius083** [comment]
Hi, 
have you solved the problem? I got similar trend for validation cost on cifar10 and cifar100(increase as learning rate decrease)
